api/enterprise-api-ref/reset-password.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: Reset password API
+published: true
+---
+
+## Request reset password link and token
+
+This call initiates the password reset process by sending a reset password
+link and token to the user's registered email address. User can click the link from the email
+and set a new password in the Mission Portal or invalidate the request.
+Every request has an expiration time equal to 48 hours.
```

Comment:
I am pretty sure that for markdown we can use one-sentence-per-line which is easier to edit/manipulate/see changes.

```suggestion
This call initiates the password reset process by sending a reset password link and token to the user's registered email address. User can click the link from the email and set a new password in the Mission Portal or invalidate the request.
Every request has an expiration time equal to 48 hours.
```